.gitignore

```diff
@@ -13,7 +13,7 @@ _build
 *.pdb
 bin
 obj
-packages
+./packages
```

Comment:
yes, in this PR, I can rename it, but is the templates related to the building of the pacakges.

Reply:
We create `packages` directories that shouldn't be in `git` in other places, so with this change as it stands, those other directories need to be found and the corresponding `.gitignore` files added/updated to include `packages` to avoid unnecessary dirty working directories.

Reply:
Issue https://github.com/xamarin/maccore/issues/2359 (has already a PR).